stdlib/test/builtin/test_list.mojo

```diff
@@ -39,7 +39,28 @@ fn test_repr_list() raises:
     assert_equal(empty.__repr__(), "[]")
 
 
+fn test_equality() raises:
```

Comment:
```suggestion
fn test_contains() raises:
```

I think calling this `test_contains` is more explicit than "equality" as we're not testing `__eq__` and `__ne__` on `ListLiteral` or `List` here, for example.